dev/run-tests.py

```diff
@@ -55,6 +55,9 @@ def determine_modules_for_files(filenames):
     for filename in filenames:
         if filename in ("appveyor.yml",):
             continue
+        if (os.environ.get("AMPLAB_JENKINS") and
+                filename in (".github/workflows/build_and_test.yml",)):
+            continue
```

Comment:
Shall we also include `test_report.yml` and `stale.yml`?

Reply:
Sure I'll add them.